lib/ansible/module_utils/network/iosxr/iosxr.py

```diff
@@ -389,7 +392,10 @@ def get_oper(module, filter=None):
 
     if filter is not None:
         try:
-            response = conn.get(filter)
+            if is_netconf(module):
+                response = conn.get(filter, remove_ns=True)
```

Comment:
The issue is fixed on devel, however, I will change as you suggested since same iosxr plugin will be copied in yang role that is currently dependent on 2.7 Ansible version.